geometry_msgs/Point[] points
string gen_map_name
---
string status
